make/istio/stop

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/make/include/has_namespace"
+source "${GIT_ROOT}/make/include/has_release"
+source "${GIT_ROOT}/make/include/defaults"
+
+printf "Stopping and cleaning up %s release...\\n" "${ISTIO_HELM_RELEASE}"
+
+if has_release "${ISTIO_HELM_RELEASE}"; then
+  helm delete --purge "${ISTIO_HELM_RELEASE}"
+fi
+
+if has_namespace "${ISTIO_NAMESPACE}"; then
+  kubectl delete namespace "${ISTIO_NAMESPACE}"
```

Comment:
Do we need to wait for the NS to actually go away? We usually do elsewhere right?